apps/webpack.config.js

```diff
@@ -75,6 +84,43 @@ const nodePolyfillConfig = {
       process: 'process/browser',
       timers: 'timers-browserify',
     }),
+    new CircularDependencyPlugin({
+      // exclude detection of files based on a RegExp
+      exclude: /node_modules|build/,
+      // add errors to webpack instead of warnings
+      failOnError: true,
+      // allow import cycles that include an asyncronous import,
+      // e.g. via import(/* webpackMode: "weak" */ './file.js')
+      allowAsyncCycles: false,
+      // set the current working directory for displaying module paths
+      cwd: process.cwd(),
```

Comment:
rather than using comment to explain what each of these settings does, can we instead use them to explain _why_ we're setting them to the specific values we are?